openapi.yaml

```diff
@@ -280,4 +294,116 @@ components:
         type: string
         example: Team A
       example: ["All teams", "Team A", "Team B"]
-
+    EnvironmentsGetResponse:
+      type: object
+      properties:
+        id:
+          title: id
+          description: Environment identifier
+          type: string
+          example: "1"
+        name:
+          title: Name
+          description: Environment name
+          type: string
+          example: DEV
+        type:
+          title: Type
+          description: Environment type
+          type: string
+          enum: [kafka, kafkaconnect, schema]
```

Comment:
Can there also be an example, to be consistent? 
Also, is there a reason the `enum` are small caps here and all caps for env status?

Reply:
The enums are defined in the API, so we can only redefine those values in here as they are in the Java application code.